tests/test_herg_qc.py

```diff
@@ -55,8 +56,8 @@ def setUp(self):
             if vend == vstart
         ]
 
-        plot_dir = "test_output"
-        os.makedirs(plot_dir, exist_ok=True)
+        plot_dir = directory_builder.setup_output_directory("test_output",
```

Comment:
These lines, @joeyshuttleworth ! If the above works just as well, then let's stick with that! No need to make the tests use a function only the scripts use. The less stuff to untangle the better